core-bundle/src/File/Metadata.php

```diff
@@ -38,11 +38,21 @@ class Metadata
     private $values;
 
     /**
-     * @param array<string, mixed> $values
+     * JSON-LD data where the key matches the schema.org
+     * type.
+     *
+     * @var array<string, array>
+     */
+    private $jsonLd;
```

Comment:
The `Metadata::$jsonLd` property is now hardwired with schema.org. Should we change this to something like the following to be open to support other schemas in the future?

```
[
	{
		"@type": "https://schema.org/MediaObject",
		"name": …,
		"caption": …,
		/…
	},
	…
]
```

Reply:
That would make things quite a bit more complicated for something we don't know if it'll ever happen. schema.org has been around since 2011 and there's no sign whatsoever for an alternative to pop up. We can still implement that when the day comes (by adding the requested schema to as an argument to the `getJsonLd()` method or so.